help/destinations/catalog/advertising/snap-inc.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Snap Inc
+description: Use your audience segments in the Snapchat Ads Platform.
+---
+
+# Snap Inc
+
+## Overview {#overview}
+
+[Snapchat Ads](https://forbusiness.snapchat.com/) are made for every business, no matter the size or industry. Become a part of Snapchatters' everyday conversations with full-screen, digital ads that inspire action from the people that matter most to your business.
+
+>[!IMPORTANT]
+>
+>This documentation page was created by the *Snap Inc* team. For any inquiries or update requests, please contact them directly at *dev-support@snap.com*
+
+## Use cases {#use-cases}
+
+This destination allows marketers to import user segments that they built using the Adobe platform into Snapchat Ads and use them to target their ads. 
+
+## Prerequisites {#prerequisites}
+
+To use this destination, you must have a Snapchat Ads account. Please refer to this documentation for information about how to create one:
+
+[Get Started with Snapchat Advertising](https://businesshelp.snapchat.com/s/article/overview?language=en_US)
+
+## Limitations {#limitations}
+
+* Snap Inc does not support multiple identities for a given Audience Segment. Please map only one identity when activating a segment.
```

Comment:
why are we using uppercase for Audience Segment? If that's a proprietary term in Snap, that's totally fine, just curious to find out.